cicecore/cicedynB/general/ice_forcing.F90

```diff
@@ -4394,7 +4153,7 @@ subroutine ocn_data_ncar(dt)
         do j = 1, ny_block 
           do i = 1, nx_block 
             if (n == 2) sss    (i,j,:) = c0
-            if (n == 3) hmix   (i,j,:) = c0
+            if (n == 3) hmix   (i,j,:) = c20
```

Comment:
Is this just temporary because of NaNs in the file?  If not, then it would be better to set this as a parameter at the top of the module, to make it more explicit.  'default mixed layer depth'

Reply:
I think this avoids a divide by zero.  I assume there are also some science reasons for setting it to 20.  I can create a parameter in the code  as suggested and will do that later today.  I think that's a good idea.

Reply:
This is a way of avoiding divison by zero.